source/frontend/src/features/disposition/list/DispositionSearchResults/DispositionSearchResults.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from 'react';
+
+import { Table } from '@/components/Table';
+import { TableSort } from '@/components/Table/TableSort';
+
+import { DispositionSearchResultModel } from '../models';
+import { columns } from './columns';
+
+export interface IDispositionSearchResultsProps {
+  results: DispositionSearchResultModel[];
+  totalItems?: number;
+  pageCount?: number;
+  pageSize?: number;
+  pageIndex?: number;
+  sort?: TableSort<DispositionSearchResultModel>;
+  setSort: (value: TableSort<DispositionSearchResultModel>) => void;
+  setPageSize?: (value: number) => void;
+  setPageIndex?: (value: number) => void;
+  loading?: boolean;
+}
+
+/**
+ * Component that renders search results for disposition files.
+ * @param {IDispositionSearchResultsProps} props
+ */
+export const DispositionSearchResults: React.FC<IDispositionSearchResultsProps> = props => {
+  const { results, sort = {}, setSort, setPageSize, setPageIndex, totalItems, ...rest } = props;
+
+  // This will get called when the table needs new data
+  const updateCurrentPage = useCallback(
+    ({ pageIndex }: { pageIndex: number }) => setPageIndex && setPageIndex(pageIndex),
+    [setPageIndex],
+  );
+
+  return (
+    <Table<DispositionSearchResultModel>
```

Comment:
the cause of the pagination issue may be that you are missing the manual pagination flag.